src/types/registration.types.ts

```diff
@@ -77,7 +77,7 @@ interface RegistrationPublisher {
 }
 
 interface AdditionalIdentifier {
-  sourceName: 'Cristin' | 'SCOPUS' | 'ISI';
+  sourceName: 'Cristin' | 'ScopusIdentifier' | 'ISI';
```

Comment:
Akkurat nå er verdien her `scopusIdentifier`, men har snakket med @truhacevkir om å endre dette til `ScopusIdentifier` for å følge samme navnekonvensjon som `Cristin`. Når det er løst vil Scopus ID vises som forventet i frontend

Reply:
Update: denne blir hetende `Scopus`: https://github.com/BIBSYSDEV/nva-publication-api/commit/abe4a3e2f7501360072e942f3ba6b6af1bec17e2